From b9f30993c88d9057d7d95a1b059989f7853fd1b0 Mon Sep 17 00:00:00 2001
From: Mattia Verga <mattia.verga@tiscali.it>
Date: Mon, 20 Jan 2025 09:04:12 +0100
Subject: [PATCH] Fix for GCC15/c++23

GCC 15 default standard is C++23. This include is needed to build zxcvbn with that language level.
---
 dict-generate.cpp | 1 +
 1 file changed, 1 insertion(+)

Index: zxcvbn-c-2.5/dict-generate.cpp
===================================================================
--- zxcvbn-c-2.5.orig/dict-generate.cpp
+++ zxcvbn-c-2.5/dict-generate.cpp
@@ -22,6 +22,7 @@
  *
  **********************************************************************************/
 
+#include <cstdint>
 #include <iostream>
 #include <string>
 #include <fstream>
